tests/test_hookman_generator/test_generate_plugin_template/expected_build_linux.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
There should be a single file file capable of building the plugin on windows vs linux.

Reply:
- [x] Fixed